tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -399,6 +399,14 @@ protected override bool Skip (Type type, string protocolName)
 				case "SRVisit":
 				case "SRWebUsage":
 					return true;
+				// iOS 14 beta 3 (to be reviewed)
+				case "GCDeviceBattery":
+				case "GCDeviceLight":
+				case "GCDualShockGamepad":
+				case "GCKeyboard":
+				case "GCMouse":
+				case "GCXboxGamepad":
```

Comment:
Is this and the one below filled as an issue somewhere? So we do not forget to review these in the future or you could check for a specific Xcode Version in the tests so they fail if not fixed in the future, also if you consider this to be an Apple mistake file a bug :)